src/Summary/SummaryForm.js

```diff
@@ -67,11 +78,11 @@ class SummaryForm extends Component {
           xs={12}
           md={6}
         >
-          <Field
-            component={Checkbox}
+          <Checkbox
```

Comment:
I wouldn't do this, since it's a form and it's logical to use `Field` of redux-form. What is the reason to switch on plain component? I don't see any

Reply:
IMHO, you can use `onChange` here, as everywhere

Reply:
Because I can't Intercept the change in value before it changed in redux-store. When I use `Field` we change the value before asking for confirmation

Reply:
> IMHO, you can use `onChange` here, as everywhere

`onChange` call after change value in redux-store.

Reply:
Right, however what stops you showing confirmation on `Field`'s `onChange` ? I see nothing. In case user confirms - just make your logic, but in case user cancels - move value back to checked.

Reply:
Ok. I will change

Reply:
agreed, plus why do you render modal on pane level and not together with field it requires?

Reply:
> agreed, plus why do you render modal on pane level and not together with field it requires?

Hmmm. I don't know. I accepted  this decision. This is problem? I don't see big difference if I insert `confirmationModal` in `SummerForm` or `PaneDetails`

Reply:
difference in components, imo modal is part of component related to the field

Reply:
ok. it makes sense